configure.py

```diff
@@ -1489,8 +1489,8 @@ def main():
                 'with_gcp_support', True, 'gcp')
   set_build_var(environ_cp, 'TF_NEED_HDFS', 'Hadoop File System',
                 'with_hdfs_support', True, 'hdfs')
-  set_build_var(environ_cp, 'TF_NEED_S3', 'Amazon S3 File System',
-                'with_s3_support', True, 's3')
```

Comment:
Is the `TF_NEED_S3` option documented anywhere, and if so can you please update it?

Reply:
@mrry I searched through but couldn't find documentation related.